### ISSUE TITLE
Migrate Azure table store from old to new structure

### ISSUE THREAD

**Issue by okms** [Puzzlepart/did#395]
### TLDR
As detailed in issues #353  #354 , azure table store has been refactored so we are now separating confirmed weeks from time entries as well as using more logical parition and row keys

We now have to migrate our [production environment](https://did365.puzzlepart.com) from the old structure into the new one.

### Description
[PMI DID365]

## Did365 boron release 

https://github.com/Puzzlepart/did365/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Milestone+5%3A+Boron%22
https://github.com/Puzzlepart/did365/issues/395

### Before D-day
- [x] Sette opp deploy timeslot
- [x] Yammer announcement, downtime 
- [x] Lag ny storage account for prod Did365 PROD
  - [x] did-85de37e8af91
  - [x] DEV-did-85de37e8af91

### On D-Day
- [x] Sette Did "down for maintenance"
- [x] eksportere - @damsleth 
  - [x] Customers
  - [x] Projects
  - [x] Roles - N/A, creating new
  - [x] Users
  - [x] ConfirmedTimeEntries 
- [x] Excel massere (endret bruk av rowkey bla) - @okms 
  - [x] Customers
  - [x] Projects
  - [x] TimeEntries
- [x] kjøre script for generering av ConfirmedPeriods fra TimeEntries @olemp 
- [x] importere til ny storage account
  - [x] ConfirmedPeriods*
  - [x] Customers*
  - [x] Labels
  - [x] Projects*
  - [x] Roles
  - [x] Users
- [x] Did back online

### Related issues & PRs
#353  
#354
#254 

### Related assets
[395-assets.zip](https://github.com/Puzzlepart/did365/files/4804907/395-assets.zip)
